MDANSE/Src/MDANSE/Framework/Jobs/DensityOfStates.py

```diff
@@ -175,6 +181,7 @@ def initialize(self):
                 (instrResolution["n_romegas"],),
                 axis="romega",
                 units="au",
+                dtype=np.complex128,
```

Comment:
I don't think we need to store the DOS (also PPS and IR) as complex.

The DOS is an FT of a symmetrised version of the VACF signal so that it becomes both real and symmetric. The DOS is a FT of a real and symmetric function so that it will always be real. The only time it wouldn't be real would be if we calculated the VACF for times -t/2 to t/2 instead of calculating it from 0 to t and symmetrising to get a VACF from -t to t.